spectator-ext-sandbox/src/main/java/com/netflix/spectator/sandbox/HttpRequestBuilder.java

```diff
@@ -231,9 +231,10 @@ public HttpResponse send() throws IOException {
         // debatable, but we have seen them in some cases if there is a high latency for
         // DNS lookups. So for now assume all exceptions are transient issues.
         if (attempt == numAttempts) {
+          LOGGER.warn("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
           throw e;
         } else {
-          LOGGER.warn("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
+          LOGGER.info("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
```

Comment:
The sandbox client shouldn't get used anymore and is only still around just in case for backwards compatibility, we don't need to be maintaining it or pushing new changes there. In general the sandbox sub-project is to incubate things and there is no guarantee of stability.

Reply:
Very good, will back out that part of the change.